molgenis/molgenis-emx2
# fix(catalogue): fix resource detail view to use correct type; fix catalogue about page should show network page but navigate as about

What are the main changes you did:
- if on a resource type path, use that type in resource detail view (e.g. 'cohorts' results in 'Cohort study'
- make 'about' path used on network page return an about page but using the resource page

how to test:
- navigate to a cohort study, and see that the breadcrumbs say 'Cohort studies'
- navigate to a catalogue, and then click the 'about'

todo:
- [ ] added/updated testplan <= @hslh can you add this?
